Pal/lib/crypto/adapters/mbedtls_adapter.c

```diff
@@ -101,6 +101,58 @@ int lib_AESCMAC(const uint8_t *key, uint64_t key_len, const uint8_t *input,
                                input, input_len, mac);
 }
 
+int lib_AESCMACInit(LIB_AESCMAC_CONTEXT * context,
```

Comment:
It might be good to add a comment here that one lib_AESCMAC is equal to an init+update*+finish sequence.